schemas/derivatives/schema.json

```diff
@@ -46,13 +48,18 @@
     {
       "label": "LP Snapshot",
       "table_name": "lp_snapshot",
-      "aggregation": "none",
+      "aggregation": "daily",
+      "uniqueKey": ["lp_address", "pool_address", "token_index", "days"],
       "description": "Liquidity providers snapshot (one entry for each token in a pool).",
       "properties": {
         "timestamp": {
           "description": "The timestamp of the snapshot.",
           "type": "number"
         },
+        "days": {
```

Comment:
whats the significance of this. why cant we just get block date instead of days

Reply:
I thought about that. Should we get rid of `days` and have protocols give us `block_date` instead?

Reply:
yes, i will prefer block date and am guessing calculating days needs block date too , so it should be easier for them to just give block date?